pages/linux/spectre-meltdown-checker.md

```diff
@@ -0,0 +1,24 @@
+# spectre-meltdown-checker
+
+> Spectre and Meltdown mitigation detection tool.
+> More information: <https://manned.org/spectre-meltdown-checker.1>.
+
+- Check the currently running kernel for Spectre or Meltdown:
+
+`spectre-meltdown-checker`
```

Comment:
Without `sudo` I get this warning:
```
Note that you should launch this script with root privileges to get accurate information.
We'll proceed but you might see permission denied errors.
To run it as root, you can try the following command: sudo /usr/bin/spectre-meltdown-checker
```
So, maybe we should add `sudo` to the commands. What do you think @bl-ue ?

Reply:
Yeah, make sense to me.

Reply:
I completely overlooked that. It is even mentioned in the Github repository.